app/lib/actions/katello/content_view/publish.rb

```diff
@@ -4,6 +4,7 @@ module ContentView
       class Publish < Actions::EntryAction
         middleware.use Actions::Middleware::KeepCurrentUser
 
+        # rubocop:disable MethodLength
```

Comment:
Hmm... I'm not sure I have a good answer. My concern is that adding a disable will mean that the next person who adds more lines to the method won't hit the warning and thus they may not care or even notice the disable (e.g. here we are adding more lines to ContentView::Remove 7aa362ca546252643accbb5f4536c2a7c9ccd32e, d3d620f971a27b6b0c60abacfd5d7c66a8fe1a60). On the other hand, my thinking behind disabling MethodLength check at a higher level or policy-wise is that this is procedural code and thus it should maybe be exempt from MethodLength checking.